css/index.css

```diff
@@ -0,0 +1,125 @@
+
+
+body 
+{background-color: #deb887;
+ text-align: left; /*is this defaut*/
+ font-family: "Oswald", sans-serif;
+ padding: 40px;}
```

Comment:
Stylistically this is a bit hard to read... Typically, you want to write your styles with this indentation:

```
body {
  background-color: #deb887;
  text-align: left;
  font-family: "Oswald", sans-serif;
  padding: 40px;
}
```

Reply:
done